meerk40t/gui/scene/scenewidgets.py

```diff
@@ -987,7 +981,7 @@ def process_draw(self, gc):
                     ends.append((x, h - length - edge_gap))
 
                     # gc.DrawText("%g %s" % (mark_point, p.units_name), x, 0, -tau / 4)
-                    gc.DrawText("%g" % mark_point, x, edge_gap, -tau / 4)
+                    gc.DrawText("%g" % mark_point, x, edge_gap, - math.tau / 4)
```

Comment:
This code is 3.6 code. We currently still support 3.5 but don't really need to.

Reply:
This can be corrected if you want to edit main.py to change to throwing the error at 3.6 instead of 3.5. We'd get access to fstrings (which are great but do not translate), underscores in binary numbers, and tau.

Reply:
I did check this before I removed it - the Pypi config says Py3.6 upwards.

Indeed, both Py3.5 has been out of support for almost 2.5 years and Py3.6 has been out of support for over a year.

If you want I will switch it to math.pi/2.

Reply:
> This can be corrected if you want to edit main.py to change to throwing the error at 3.6 instead of 3.5. We'd get access to fstrings (which are great but do not translate), underscores in binary numbers, and tau.

I will do this. Personally I have no issues with allowing 3.6 even though it is unsupported.